test/end-to-end/stock/stock.z2.inventory-adjustment-inventories.spec.js

```diff
@@ -25,7 +25,7 @@ function StockInventoriesRegistryTests() {
   it('find 3 inventory in Depot Principal plus one line for the Grouping', async () => {
     await modal.setDepot('Depot Principal');
     await modal.submit();
-    await GU.expectRowCount(gridId, 2 + GROUPING_ROW);
+    await GU.expectRowCount(gridId, 3 + GROUPING_ROW);
```

Comment:
![image](https://user-images.githubusercontent.com/896472/93211614-66f98500-f759-11ea-9a8f-71c5f3b5378d.png)

Just FYI, when I run this on my machine, this is the state of the registry when we get to this failing test.